dragster/tslint.json

```diff
@@ -1 +0,0 @@
-{ "extends": "../tslint.json" }
```

Comment:
It failed because of the multiple module export. I saw some other library use that way. Anyway, I can put it back, @DanielRosenwasser solved that problem.